Makefile

```diff
@@ -129,8 +141,34 @@ docs-in-docker:
 
 ci: check test
 
+$(VM_IMAGE): TAG=HEAD
+$(VM_IMAGE): archive bots
```

Comment:
I think this needs to depend on `srpm`, not `archive`?

Reply:
Yeah, indeed. Worked for me because I didn't delete it between runs.